src/main/java/seedu/workbook/model/internship/Stage.java

```diff
@@ -43,15 +44,8 @@ public static boolean isValidStage(String test) {
     /**
      * Returns true if a given stage has curated tips.
      */
-    public static boolean hasNoTips(Stage stage) {
-        return !stageWithTips.contains(stage);
-    }
-
-    /**
-     * Returns list of stages with curated tips.
-     */
-    public static String stageWithTipsToString() {
-        return stageWithTips.toString();
+    public boolean hasNoTips() {
+        return !StageUtil.getStagesWithTips().contains(this);
```

Comment:
violates "Tell, Don't Ask" principle, should define a method in StageUtil to check `this`